src/display/canvas.js

```diff
@@ -144,6 +151,12 @@ function addContextCurrentTransform(ctx) {
       ctx._originalSetTransform(a, b, c, d, e, f);
     };
 
+    ctx.resetTransform = function ctxResetTransform(a, b, c, d, e, f) {
```

Comment:
Given that this, and all multiplications below now happen unconditionally for every PDF even though this is a Chrome-specific issue, I'd really like to see a benchmark here to make sure performance of larger PDF files from the test suite doesn't suffer. See https://github.com/mozilla/pdf.js/wiki/Benchmarking-your-changes for how to do this.

Reply:
I think this is fine, since it does *not* apply to Firefox given that all of this code is within the `if (!ctx.mozCurrentTransform) { ... }` block

Reply:
The stuff here is chrome-only so there is no impact for firefox.